internal/tags/tagpolicy/tagris-cfn-terraform-mapping.csv

```diff
@@ -0,0 +1,646 @@
+CFN Alias,Tag Policies Alias,Terraform Resource Type
```

Comment:
Do we get this as-is from AWS? Why CSV vs. HCL?

Reply:
CSV can be such a pain to edit and git diffs don't work well so you get more merge conflicts